IdentityCore/src/controllers/ios/MSIDBrokerInteractiveController.m

```diff
@@ -97,6 +99,26 @@ - (void)acquireToken:(nonnull MSIDRequestCompletionBlock)completionBlock
         return;
     }
     
+    NSString *upn = self.interactiveParameters.accountIdentifier.displayableId ?: self.interactiveParameters.loginHint;
+    
+    [self.interactiveParameters.authority resolveAndValidate:self.interactiveParameters.validateAuthority
+                                           userPrincipalName:upn
+                                                     context:self.interactiveParameters
+                                             completionBlock:^(__unused NSURL *openIdConfigurationEndpoint,
+                                                               __unused BOOL validated, NSError *error)
+     {
+         if (error)
```

Comment:
I also saw this in other requests, but why are we not checking validated?

Reply:
Because if validation is OFF, this flag might come as NO. If validation is ON and it doesn't get validated, we'll get an error back. So, the internal functionality already verifies that if validate is YES, error would be filled in.